codeship-steps.yml

```diff
@@ -6,5 +6,13 @@
   steps:
     #-  command: ./.ci/travis/travis.sh site
     #-  command: ./.ci/travis/travis.sh nondex
-    -  command: ls -la
-    -  command: .ci/run-link-check-plugin.sh
+    - command: |
+        ls -la
+        SKIP_FILES1=".github|distelli-manifest.yml|fast-forward-merge.sh"
+        SKIP_FILES2="LICENSE|LICENSE.apache20|README.md|release.sh|RIGHTS.antlr"
+        SKIP_FILES3="shippable.yml|wercker.yml|appveyor.yml|circle.yml"
+        SKIP_FILES="$SKIP_FILES1|$SKIP_FILES2|$SKIP_FILES3"
+        if [[ $(git diff --name-only HEAD HEAD~1 | grep -vE "$SKIP_FILES" | cat | wc -c) > 0 ]];
```

Comment:
@romani Is this safe to do when travis and other CIs dropped this and forced to run all the time?
https://github.com/checkstyle/checkstyle/blob/master/.travis.yml#L326

Reply:
it is not safe, but till it present in other CI, lets keep it here too.
Removal of all is another issue, as far as I remember we did not decided on it yet.